pegasus/helpers/form_helpers.rb

```diff
@@ -93,6 +89,7 @@ def update_form(kind, secret, data)
 
   prev_data = JSON.parse(form[:data], symbolize_names: true)
   symbolized_data = Hash[data.map {|k, v| [k.to_sym, v]}]
+  symbolized_data.merge!(Hash[JSON.parse(symbolized_data[:data]).map {|k, v| [k.to_sym, v]}])
```

Comment:
I think it would be a whole lot more readable to leverage ActiveSupport's [Hash.deep_symbolize_keys!](https://apidock.com/rails/v4.2.7/Hash/deep_symbolize_keys%21) (or `deep_symbolize_keys`) instead of this cryptic `Hash[ data.map {|k, v| [k.to_sym, v]}]` pattern.

```ruby
require 'active_support/core_ext/hash'
# ...
symbolized_data = data.deep_symbolize_keys
```

Next, I'm confused by this new line. Even if we change it to:
```ruby
symbolized_data.merge! symbolized_data[:data].deep_symbolize_keys
```
Why is `:data` contained in `symbolized_data`? Is the outer data the entire form, and the `data` field is the JSON blob representing the specific form data? In that case, maybe we should call the outer one `form` or `row` instead of `data`?